### DISCUSSION TITLE
How to map mime type

### DISCUSSION THREAD

**Discussion by siddharth223437** [micronaut-projects/micronaut-core#7301] [General] [answered]
Micronaut think that all ".mjs" file are "text/plain" so how can i tell micronaut to map all ".mjs" to "application/javascript". Below is  springboot code:

```
 @Override
    public void customize(ConfigurableServletWebServerFactory factory) {
        MimeMappings mappings = new MimeMappings(MimeMappings.DEFAULT);
        mappings.add("mjs", "application/javascript");
        factory.setMimeMappings(mappings);
    }
```

**Comment by yawkat** [comment, marked as answer]
I don't think there's a supported way to do this atm. You could try amending this file with your type, and putting it into `META-INF/http/mime.types` https://github.com/micronaut-projects/micronaut-core/blob/b5cc40ea85d020d875f5a95e570188d4a47890a1/http/src/main/resources/META-INF/http/mime.types#L7
